### ISSUE TITLE
STRUCT does not allow a FieldName of 'Properties'

### ISSUE THREAD

**Issue by mbrancato** [confluentinc/ksql#1796]
I'm not sure how this came to be, but it was an odd error I ran into. From some guessing, it seems a workaround is to place it in double quotes. I haven't tested if this functions properly when accessing a child object.
`"Properties"`

Example query below:

```
CREATE STREAM test(
field0 varchar,
data  struct<item0 varchar, Properties varchar, somethingelse varchar>,
id int )
WITH (KAFKA_TOPIC='test', VALUE_FORMAT='json');
```

Error:

> io.confluent.ksql.parser.exception.ParseFailedException: Exception while processing statements :line 18:35: extraneous input 'Properties' expecting {'ADD', 'APPROXIMATE'....

**Comment by hjafarpour** [comment]
@mbrancato 'Properties' is a reserved word in KSQL grammar. You can use single quotes to specify a reserved word as column name.
Try this: 
```
CREATE STREAM test(
field0 varchar,
data  struct<item0 varchar, 'Properties' varchar, somethingelse varchar>,
id int )
WITH (KAFKA_TOPIC='test', VALUE_FORMAT='json');
```

Then you can write queries like this:
```
CREATE STREAM S1 AS SELECT data->`Properties` FROM TEST;
```
Note that when you have the column name in quotes it will be case sensitive.

**Comment by miguno** [comment]
Can we close this @mbrancato ?

**Comment by mbrancato** [comment]
I think I was unable to get backticks to work at the time I ran across this, IIRC. I had to use double quotes. Closing this as it seems to be known. Might be an opportunity for mentioning reserved words in the docs.

**Comment by miguno** [comment]
cc @JimGalasyn 

> Might be an opportunity for mentioning reserved words in the docs.  ([source](https://github.com/confluentinc/ksql/issues/1796#issuecomment-432889991))

**Comment by miguno** [comment]
Thanks for the follow-up, @mbrancato !

**Comment by JimGalasyn** [comment]
Tracking with internal ticket KSQL-1940.
